packages/core/src/messages/packet.ts

```diff
@@ -244,7 +235,11 @@ export class Packet {
     if (isDirectMessage) {
       ticket = channel.createDummyTicket(ticketChallenge)
     } else {
-      ticket = await channel.createTicket(ticketOpts.value, ticketChallenge, ticketOpts.winProb)
+      ticket = await channel.createTicket(
+        new Balance(new BN(TICKET_BASE_UNIT).muln(path.length - 1)),
```

Comment:
can u link the part in the DD where it describes this? shouldn't each ticket contain a value of `TICKET_AMOUNT`?

Reply:
sending you a DM

Reply:
I guess we can resolve this?

Reply:
as discussed privately, this change is not described in our DD

Reply:
I don't agree with this change as it stands, and since there is not description of this in the DD I can't approve it

Reply:
This **is** the HOPR protocol and has been there for years. Recently it got removed and was for a long time incorrectly implemented.

> since there is not description of this in the DD I can't approve it

I don't think this is base for meaningful discussion

Reply:
ehm, we just had this conversation privately and u told me it's not in the DD, how am I supposed to approve a PR if _we_ don't know how it should work?

Reply:
Can we first make sure this is in the DD, then we can confirm here?

Reply:
Omw, to improve documentation there. `TICKET_BASE_UNIT` currently is a simplification to circumvent problems with roundings and is supposed to change the functionality only in terms rounding errors

Reply:
obsolete now, using inverseProbs